### PR TITLE
this creates js files to toggle lighting, might find a way to merge into one later on

### PR DESCRIPTION
**Pull request by USMP-lancer**

### CODE REVIEW

File: dark.js [hunk 1]
```diff
@@ -0,0 +1 @@
+Vars.state.rules.lighting
```
Comment (Quezler) [inline dark.js]:
```suggestion
Vars.state.rules.lighting=false
```

**Review by Quezler — CHANGES_REQUESTED** [review]
Perhaps also include a call to sync the roles?

File: dark.js (hunk 1 above)
Comment (USMP-lancer) [reply dark.js]: see im a genius and never make mistakes abtuabsgakjbgl

**Comment by Quezler** [comment]
it should probably sync as well

**Comment by USMP-lancer** [comment]
lighting auto syncs actually, so the sync it not really needed

**Comment by USMP-lancer** [comment]
i knwo because i tested on nydus

**Comment by USMP-lancer** [comment]
im lancer btw, i just have this namd bvcbvc for some reason, im gonna change the name if i can

**Comment by USMP-lancer** [comment]
now my user is this

**Comment by Quezler** [comment]
> lighting auto syncs actually, so the sync it not really needed

that makes little sense, but fine

**Pull request merged**